hs/src/Reach/AST/SL.hs

```diff
@@ -1,4 +1,5 @@
 {-# OPTIONS_GHC -Wno-missing-export-lists #-}
+{-# Language DefaultSignatures #-}
```

Comment:
What does this do? Should we add it to the standard list?

Reply:
it allows line 167 but will probably delete this anyway based on the comment below this